scipy/interpolate/_bsplines.py

```diff
@@ -620,6 +632,14 @@ def make_interp_spline(x, y, k=3, t=None, bc_type=None, axis=0,
         be an iterable of pairs ``(order, value)`` which gives the values of
         derivatives of specified orders at the given edge of the interpolation
         interval.
+        Alternatively, the following string aliases are recognized:
+        * ``"clamped"``: The first derivatives at the ends are zero. This is
+           equivalent to ``bc_type=((1, 0.0), (1, 0.0))``.
+        * ``"natural"``: The second derivatives at ends are zero. This is
+          equivalent to ``bc_type=((2, 0.0), (2, 0.0))``.
+        * ``"not-a-know"`` (default): The first and second segments are the same
```

Comment:
`"not-a-knot"` I imagine.

Reply:
Certainly. Fixed and force-pushed. Thanks!